src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -94,6 +108,49 @@ public LoanRepository(Clients clients) {
       .thenComposeAsync(r -> r.after(this::refreshLoanRepresentation));
   }
 
+  public CompletableFuture<Result<LoanAndRelatedRecords>> replaceLoan(Loan loan) {
```

Comment:
Is it completely new logic or is it based on the "single" loan update?

Reply:
It's been moved from a single loan update